spec/ysh-builtins.test.sh

```diff
@@ -629,3 +629,31 @@ echo $[l2[1]] # NOTE: depends on dict being ordered
 3
 foo
 ## END
+
+#### List->extend()
+var l = List(1:3)
+echo $[len(l)]
+_ l->extend(List(3:6))
+echo $[len(l)]
+## STDOUT:
+2
+5
+## END
+
+#### List mutators should return null
+shopt -s oil:all
+var l = List(1:3)
+
+var result = l->extend(List(3:6))
+if (result === null) { echo 'pass' }
+
+setvar result = l->append(6)
+if (result === null) { echo pass }
+
+setvar result = l->pop()
```

Comment:
Actually pop() is the only one that returns a value!   

```
>>> ['a', 'b'].pop()
'b'
```

I guess the original code made that mistake too ..?

Reply:
Done